Ghidra/Features/GraphServices/src/main/java/ghidra/graph/visualization/DefaultGraphDisplay.java

```diff
@@ -229,7 +229,7 @@ private void createActions() {
 				.toolBarIcon(Icons.REFRESH_ICON)
 				.onAction(context -> {
 					viewer.reset();
-					viewer.scaleToLayout();
+					viewer.scaleToLayout(true);
```

Comment:
passing 'true' will cause the current vertex positions to be considered when making the graph view fit the window